commitlint.config.ts

```diff
@@ -0,0 +1,25 @@
+import { RuleConfigSeverity, type UserConfig } from "@commitlint/types";
+
+const Configuration: UserConfig = {
```

Comment:
Type safe commitlint configuration! Hooray!

Reply:
Nope, this wasn't worth it. In isolate it's a nice idea, but it required making some other stuff uglier.